### PR TITLE
fixes: NGO Open Data

### PR DESCRIPTION
**Pull request by OlteanuCosmin**
- C4R-418: limit input to 6 characters
- C4R-419: only positive numbers should be allowed
- C4R-420: missing error when decimal present
- C4R-424: redirected to wrong page
- C4R-425: missing error on invalid URL
- C4R-426: missing error on invalid file format

### CODE REVIEW

File: frontend/src/pages/organization/components/OrganizationData/OrganizationData.tsx [hunk 1]
```diff
@@ -160,6 +160,10 @@ const OrganizationData = () => {
   const onSaveReport = (data: Report) => {
     setIsActivitySummaryModalOpen(false);
     setSelectedReport(null);
+    if (data.report?.startsWith('www')) {
```
Comment (radulescuandrew) [inline frontend/src/pages/organization/components/OrganizationData/OrganizationData.tsx:163]: Fix din Front-end. Addon-ul din frontend (Https) ar trebui sa puna valoare si sa trimita catre Backend, nu facem validarea aici

File: frontend/src/pages/organization/components/OrganizationData/OrganizationData.tsx [hunk 2]
```diff
@@ -215,6 +219,11 @@ const OrganizationData = () => {
   };
 
   const uploadPartnersList = async (partnerId: number, file: File) => {
+    if (!isExcel(file.name)) {
```
Comment (radulescuandrew) [inline frontend/src/pages/organization/components/OrganizationData/OrganizationData.tsx:222]: @birloiflorian readXlsxFile citeste si CSV-uri sau doar Excel?

Ar trebui sa avem un try/catch pe readXlsxFile sa parseze eroarea si sa afiseze eroarea

**Review by radulescuandrew — CHANGES_REQUESTED** [review]
@birloiflorian mai arunca o privire pe files te rog

File: frontend/src/pages/organization/components/OrganizationData/OrganizationData.tsx (hunk 1 above)
Comment (radulescuandrew) [reply frontend/src/pages/organization/components/OrganizationData/OrganizationData.tsx:163]: @dragos1195 uite-te tu te rog. Ar trebui sa fie concatenata valoarea `https://`

Comment (dragos1195) [reply frontend/src/pages/organization/components/OrganizationData/OrganizationData.tsx:163]: https://wearetribus.atlassian.net/browse/C4R-466 Created a task for this. Should leave it like this for now so the testing will pass. Afterwards we'll change how the "http(s)" is appended.

**Pull request merged**